packages/web-runtime/src/composables/head/useHead.ts

```diff
@@ -1,20 +1,26 @@
-import { computed } from 'vue'
+import { computed, unref } from 'vue'
 import { Store } from 'vuex'
 import { useHead as _useHead } from '@vueuse/head'
 import { getBackendVersion, getWebVersion } from 'web-runtime/src/container/versions'
+import { useThemeStore } from '@ownclouders/web-pkg'
+import { storeToRefs } from 'pinia'
 
 export const useHead = ({ store }: { store: Store<any> }) => {
+  const themeStore = useThemeStore()
+  const { currentTheme } = storeToRefs(themeStore)
+
   _useHead(
     computed(() => {
-      const favicon = store.getters['configuration']?.currentTheme?.logo?.favicon
+      const favicon = computed(() => currentTheme.value.logo.favicon)
```

Comment:
Computed inside computed? 🤔 Not sure how it behaves, but we should clean this probably.